Apply rustfmt

jncraton opened the pull request:
There are no substantive changes here. I'm playing around with a branch, and I was starting the process of cleaning up my code when I thought that it might be better to just use rustfmt. All I did was checkout the code from master and run it through the rustfmt-nightly.

If this isn't desired, feel free to close this PR.

gchp commented:
Definitely desired! However I've just introduced a bunch of conflicts with this PR. Could you either rebase or re-submit off the latest master?

jncraton commented:
I've rebased this. I also changed the build over to the nightly toolchain to get the latest rustfmt and because the master branch build no longer succeeds on stable with your latest commit.
